src/Aspire.Hosting.Azure.EventHubs/PublicAPI.Shipped.txt

```diff
@@ -6,5 +6,3 @@ Aspire.Hosting.Azure.AzureEventHubsResource.EventHubsEndpoint.get -> Aspire.Host
 Aspire.Hosting.AzureEventHubsExtensions
 static Aspire.Hosting.AzureEventHubsExtensions.AddAzureEventHubs(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>!
 static Aspire.Hosting.AzureEventHubsExtensions.AddAzureEventHubs(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, System.Action<Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>!, Aspire.Hosting.ResourceModuleConstruct!, Azure.Provisioning.EventHubs.EventHubsNamespace!>? configureResource) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>!
-static Aspire.Hosting.AzureEventHubsExtensions.AddEventHub(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>! builder, string! name) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>!
```

Comment:
This isn't how this is supposed to work. You don't modify the "Shipped.txt" file. Instead you put a `*REMOVED*` line in the .Unshipped.txt file.

Reply:
I have no idea how this undocumented, opaque and loathsome analyzer is meant to work. I went along with the Rosyln Analyzer's suggestion to correct the problem and here we are 😆

Reply:
You aren't alone.

cc @agocke